manifests/linux/provisioning

```diff
@@ -0,0 +1,123 @@
+copy,/var/lib/waagent/provisioned
+copy,/var/lib/waagent/waagent_status.json
+copy,/var/lib/waagent/waagent_status.*.json
+copy,/var/lib/waagent/*/error.json
+copy,/var/lib/waagent/*.manifest.xml
+copy,/var/lib/waagent/*/config/*.settings
+copy,/var/lib/waagent/*/config/HandlerState
+copy,/var/lib/waagent/*/config/HandlerStatus
+copy,/var/lib/waagent/*/status/*.status
+copy,/var/lib/waagent/ExtensionsConfig.*.xml
+copy,/var/lib/waagent/GoalState.*.xml
+copy,/var/lib/waagent/HostingEnvironmentConfig.xml
+copy,/var/lib/waagent/Incarnation
+copy,/var/lib/waagent/ManagedIdentity-*.json
+copy,/var/lib/waagent/SharedConfig.xml
+copy,/var/lib/waagent/history/*.zip,noscan
+echo,
+
+echo,### Gathering Dhcp and Dhclient Files ###
+copy,/var/lib/NetworkManager/*.lease
+copy,/var/lib/NetworkManager/*.leases
+copy,/var/lib/wicked/lease*
+copy,/var/lib/dhclient/*.lease
+copy,/var/lib/dhclient/*.leases
+copy,/var/lib/dhcp/*.lease
+copy,/var/lib/dhcp/*.leases
+echo,
+
+echo,### Gathering Networking Files ###
+copy,/etc/dhclient*.conf
+copy,/etc/netplan/*.yaml
+copy,/etc/dhcp/*.conf
+copy,/etc/network/interfaces
+copy,/etc/network/interfaces.d/*.cfg
+copy,/etc/ufw/ufw.conf
+copy,/etc/ssh/sshd_config
+copy,/etc/ssh/sshd_config.d/*.conf
+copy,/etc/nsswitch.conf
+copy,/etc/resolv.conf
+copy,/etc/idmapd.conf
+copy,/etc/chrony/chrony.conf
+echo,
+
+echo,### Gathering NetworkManager Files ###
+copy,/usr/lib/NetworkManager/*.conf
+copy,/usr/lib/NetworkManager/conf.d/*.conf
+copy,/etc/NetworkManager/*.conf
+copy,/etc/NetworkManager/conf.d/*.conf
+copy,/var/lib/NetworkManager/*.conf
+copy,/var/lib/NetworkManager/conf.d/*.conf
+copy,/var/lib/NetworkManager/*.state
+echo,
+
+echo,### Gathering Sysconfig Network Files ###
+copy,/etc/sysconfig/network/dhcp
+copy,/etc/sysconfig/network
+copy,/etc/sysconfig/network-scripts/ifcfg-*
+copy,/etc/sysconfig/network-scripts/route-*
+copy,/etc/sysconfig/network/config
+copy,/etc/sysconfig/network/ifcfg-*
+copy,/etc/sysconfig/network/routes
+copy,/etc/sysconfig/iptables
+copy,/etc/sysconfig/SuSEfirewall2
+echo,
+
+echo,### Gathering Wicked Network Files ###
+copy,/etc/wicked/*.xml
+echo,
+
+echo,### Gathering Package Management Log Files ###
+copy,/var/log/dpkg*
+copy,/var/log/yum*
+copy,/var/log/dnf*
+echo,
+
+echo,### Gathering System Log Files ###
+copy,/var/log/syslog*
+copy,/var/log/rsyslog*
+copy,/var/log/messages*
+copy,/var/log/kern*
+copy,/var/log/dmesg*
+copy,/var/log/boot*
+copy,/var/log/auth*
+copy,/var/log/secure*
+copy,/var/log/journal/*/*,noscan
+echo,
+
+echo,### Gathering Azure Files ###
+copy,/var/log/azure/*/*
+copy,/var/log/azure/*/*/*
+copy,/var/log/azure/custom-script/handler.log
+copy,/var/log/azure/run-command/handler.log
+copy,/var/log/azure/cluster-provision.log
+echo,
+
+echo,### Gathering System Configuration Files ###
+copy,/etc/fstab
+copy,/boot/grub*/grub.c*
+copy,/boot/grub*/menu.lst
+copy,/etc/*-release
+copy,/etc/HOSTNAME
+copy,/etc/hostname
+echo,
+
+echo,### Gathering Disk Info ###
+diskinfo,
```

Comment:
could you mind help add azure-proxy-agent log files for provisioning mode too? I have it at agents and normal mode already.